cmd/thanos/sidecar.go

```diff
@@ -362,7 +375,7 @@ func runSidecar(
 			s := shipper.New(logger, reg, conf.tsdb.path, bkt, m.Labels, metadata.SidecarSource,
 				uploadCompactedFunc, conf.shipper.allowOutOfOrderUpload, metadata.HashFunc(conf.shipper.hashFunc), conf.shipper.metaFileName)
 
-			return runutil.Repeat(30*time.Second, ctx.Done(), func() error {
+			return runutil.RepeatWithJitter(30*time.Second, ctx, 0.05, func() error {
```

Comment:
0.05 variance seems too small as it is only `30s * 0.05 = 1.5s`?
I am thinking about 0.2 here maybe

Reply:
Resolved all !